packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -0,0 +1,503 @@
+description: The Table is used to represent static tabular data.
+slug: /components/table/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Table, THead, TBody, TFoot, Tr, Th, Td} from '@twilio-paste/table';
+import Changelog from '@twilio-paste/table/CHANGELOG.md';
+import {Text} from '@twilio-paste/text';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/table"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/table/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/table"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-table--basic-table"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for you to use in production. Components such as Table are likely to not see any API
+    changes. This is only an alpha due to the lack of documentation and a matching Sketch symbol. Feel free to share
+    your experience using this component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new">starting a discussion on GitHub</Anchor>. We are
+    also looking for a contributor to symbolize this component in Sketch. Interested? Reach out!
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Table
+
+A Table is used to represent tabular data, or information presented in a 2-dimensional table composed of rows and columns of cells containing data.
+Tables fill 100% width of the space of their containing element.
+
+You can create tables using a combination of:
+
+- **Table:** The table wrapper.
+- **THead:** The header of your table.
+- **TBody:** The main body of your table.
+- **TFoot:** The footer of your table.
+- **Tr:** A row of cells in your table.
+- **Th:** A header cell which can be used as a column title.
+- **Td:** A cell of table content.
+
+### Accessibility
+
+Tables are used to organize data with a logical relationship in grids. Accessible tables need HTML markup that indicates
+header cells and data cells and defines their relationship. Assistive technologies use this information to provide context to users.
```

Comment:
Under here it might be nice to list out 2 must haves:

- Tables **must** have a `THead`, containing a Table Header `Th` for each column.
- Tables **must** have a `TBody` wrapping the table body of rows.